### DISCUSSION TITLE
希望外置稍后再看的按钮旁边可以显示稍后再看列表中的视频数量

### DISCUSSION THREAD

**Discussion by zbodo** [the1812/Bilibili-Evolved#1512] [功能建议 / Ideas]
既然把按钮旁边的文字给移除了，而同一排的几个按钮旁边都是有表示数目的数字的
![image](https://user-images.githubusercontent.com/39948021/106376447-67c14980-63d0-11eb-8658-07cc6da7097a.png)
希望在稍后再看按钮旁边显示已有视频数量，主要因为感觉这样会更一致、美观。功能还是其次

**Comment by Geeyun-JY3** [comment]
稍后再看的这个数字不会感觉和旁边的数字不一致吗……旁边这些数字指的是该视频的相关数据，你说的这个数字指的是用户已有视频数量的视频数。

**Reply by zbodo** [reply]
会比什么都没有好得多

**Reply by Geeyun-JY3** [reply]
不一致容易让用户迷惑的
